### PR TITLE
Locations endpoint

### LINKED ISSUE #44: RSpec test query for returning available cities/states

### PR DESCRIPTION
**Pull request by algibson1**
## Explain changes:
Added a query field called 'locations' that allows the FE to query for a list of unique locations where there are registered organizations in the database

Endpoint is also tested with RSpec

### What does this PR do?
- [X] Feature
- [ ] Bug fix
- [ ] Refactor
- [ ] Styling
- [ ] Other:

### Necessary checkmarks:
- [X] All Tests are Passing
- [ ] The code will run locally

### Testing Changes
- [ ] No Tests have been changed/added
- [X] Some Tests have been changed/added
- [ ] All of the Tests have been changed(Please explain)

## Issues this closes 
#44 
#35 
#6

### CODE REVIEW

File: spec/graphql/queries/locations_spec.rb [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+          "state" => "OR"
+        },
+        {
+          "city" => "Cheyenne",
+          "state" => "WY"
+        }
+      ]
+    }
+
+    result = RefugeeAidBeSchema.execute(query)
```
Comment (DavisWeimer) [inline spec/graphql/queries/locations_spec.rb:45]: Very nice!

Comment (DavisWeimer) [reply spec/graphql/queries/locations_spec.rb:45]: Disregard! 😁 I just looked at the `query_type.rb` and `locations` method which has no arguments and no need for `variables:`

**Review by DavisWeimer — APPROVED** [review]
Green dots, all day! 🟢

**Review by ethanrossblack — APPROVED** [review]
nice job with the endpoint!

**Pull request merged**